src/core/algorithms.js

```diff
@@ -2,11 +2,16 @@
 Currently used only for adding 'assert' class to algorithm lists  
 */
 
+import css from "text!../../assets/algorithms.css";
+
 export const name = "core/algorithms";
 
 export async function run() {
   const elements = Array.from(document.querySelectorAll("ol.algorithm li"));
   elements
     .filter(li => li.innerText.trim().startsWith("Assert: "))
     .forEach(li => li.classList.add("assert"));
+  const style = document.createElement("style");
```

Comment:
Yes, I will modify that. Also, about the whitespace fix, what utility do you use to find that it is required. Because it passes through ESlint, so how do I check for these myself?

Reply:
Ah, good point. We should add [eol-last](https://eslint.org/docs/rules/eol-last) to our ES Lint config. Been checking those manually.